pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go

```diff
@@ -45,6 +48,42 @@ func (ms *MetricSender) ReportNetworkDeviceMetadata(config *checkconfig.CheckCon
 		}
 		ms.sender.EventPlatformEvent(string(payloadBytes), epforwarder.EventTypeNetworkDevicesMetadata)
 	}
+
+	// Telemetry
+	for _, interfaceStatus := range interfaces {
+		status := computeInterfaceStatus(interfaceStatus.AdminStatus, interfaceStatus.OperStatus)
+		interfaceTags := []string{"status:" + status, "interface:" + interfaceStatus.Name, "interface_alias:" + interfaceStatus.Alias}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Should we also add an `interface_index` tag? I think this can be help in some cases.

Reply:
I was looking at other interface metrics coming from profile which don't have the interface index and did the same here. But after second thoughts I think it can't hurt to also have this tag (it should not add any new context) and maybe can prevent duplicates if the name and alias collides.

Reply:
I think we should probably also add an interface_index tag to the generic-if profile too. It can help debug, avoid duplication, more visibility.